src/core/config/remote/remote_config_tasks.ts

```diff
@@ -0,0 +1,246 @@
+/**
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the ""License"");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an ""AS IS"" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import {
+  RelayComponent, HaProxyComponent, EnvoyProxyComponent, MirrorNodeComponent, ConsensusNodeComponent,
+} from './components/index.js'
+import { ComponentType, ConsensusNodeStates } from './enumerations.js'
+import chalk from 'chalk'
+import { SoloError } from '../../errors.js'
+
+import type { Listr, ListrTask } from 'listr2'
+import type { NodeAliases } from '../../../types/aliases.js'
+import type { BaseCommand } from '../../../commands/base.js'
+import type { RelayCommand } from '../../../commands/relay.js'
+import type { NetworkCommand } from '../../../commands/network.js'
+import type { DeploymentCommand } from '../../../commands/deployment.js'
+import type { MirrorNodeCommand } from '../../../commands/mirror_node.js'
+import type { NodeCommandHandlers } from '../../../commands/node/handlers.js'
+
+/**
+ * Static class that handles all tasks related to remote config used by other commands.
+ */
+export class RemoteConfigTasks {
+
+  /* ----------- Create and Load ----------- */
+
+  /**
+   * Loads the remote config from the config class.
+   *
+   * @param argv - used to update the last executed command and command history
+   */
+  public static loadRemoteConfig (this: BaseCommand, argv: any): ListrTask<any, any, any> {
+    return this.remoteConfigManager.buildLoadTask(argv)
```

Comment:
I'm concerned about how frequently this is being called.  

Looking through handlers.ts, I see the issue you are trying to solve.  I think if you only do the load and modify in the functions that have the commandActionBuilder, then you will remove all of the duplication.